src/components/LimitReset/AllowanceRemover.tsx

```diff
@@ -68,11 +77,17 @@ export default function AllowanceRemover({
           <CustomLightSpinner src={Circle} alt="loader" size={'23px'} />
         </ButtonPrimary>
       )}
-      {isLoadingDone && maliciousAllowanceCount == 0 ? (
-        <p style={{ color: 'green' }}>
-          {' '}
-          You don&apos;t have any bad allowances on this account related to Kromatika ! 🎉🎉
-        </p>
+      {isLoadingDone ? (
+        isLoadingDone && maliciousAllowanceCount == 0 ? (
+          <p style={{ color: 'green' }}>
+            {' '}
+            You don&apos;t have any bad allowances on this account related to Kromatika ! 🎉🎉
+          </p>
+        ) : isLoadingDone && isMaliciousAllowancesRemoved ? (
+          <p style={{ color: 'green' }}> Malicious allowances have been removed! 🎉🎉</p>
+        ) : (
+          <p style={{ color: 'red' }}> An error has occured ! 🎉🎉</p>
```

Comment:
maybe this case should be empty or atleast it shouldn't display an error has occured; while the txn is not approved and mined on the blockchain (while the process is ongoing), it is displaying "an error has occured"